src/main/java/com/strandls/document/service/DocumentService.java

```diff
@@ -99,4 +103,15 @@ public String bulkUploadBibTex(HttpServletRequest request, InputStream uploadedI
 
 	public Boolean documentDownloadLog(HttpServletRequest request, DownloadLogData downloadLogData);
 
+	public void produceToRabbitMQ(String documentId, String updateType);
+	
+	
```

Comment:
create a new interface and implementation class for the list page, this interface and impl should only be working for document show page and edits in the list page and show page